packaging/hoodie-hive-bundle/pom.xml

```diff
@@ -71,7 +71,12 @@
     <dependency>
       <groupId>org.apache.thrift</groupId>
       <artifactId>libthrift</artifactId>
-      <version>0.9.2</version>
+      <version>${thrift.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.thrift</groupId>
```

Comment:
Can you please remind me why this second thrift dependency is needed ?

Reply:
This is the runtime dependency coming from Hive Metastore client. libfb303 contains  requests/response definitions. Without fixing the version, we run into NoSuchMethod errors at runtime